src/print.c

```diff
@@ -53,6 +56,21 @@ int apk_get_screen_width(void)
 	return apk_screen_width;
 }
 
+const char *apk_get_human_size(off_t size, off_t *dest)
+{
+	size_t i;
+	off_t s;
+
+	assert(size >= 0);
+
+	for (i = 0, s = size; s >= 1024 &&
+	     i < ARRAY_SIZE(apk_size_units); i++)
+		s /= 1024;
+
+	if (dest) *dest = s;
+	return apk_size_units[min(i, ARRAY_SIZE(apk_size_units))];
```

Comment:
Shouldn't this be `min(i, ARRAY_SIZE(apk_size_units) - 1)`?

Reply:
Yes.